feature-detects/css/remunit.js

```diff
@@ -25,7 +25,7 @@ define(['Modernizr', 'createElement'], function( Modernizr, createElement ) {
     try {
       div.style.fontSize = '3rem';
     }
-    catch( er ) {}
+    catch ( er ) {}
```

Comment:
Sure - I'm just using the `google` preset as a base/start so yeah feel free for more suggestions in rules to add/remove.